_posts/2021-07-02-migrating-from-legacy-drupal-to-federalist.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Migrating from Legacy Drupal to Federalist"
+date: July 2, 2021
+excerpt: "With the end of life dates for Drupal versions 7 and 8 fast approaching, this tech talk will discuss strategies for migrating from legacy versions of Drupal to the Federalist platform."
+---
+
+**Note - this blog post includes content originally presented during a cloud.gov / Federalist tech talk on June 17th, 2021.**
+
+Drupal is a commonly used content management system (CMS) in the Federal government. Many existing Drupal sites are still using older, outdated versions of the CMS and site owners may be facing challenges with upgrading to the latest version. In addition, because Drupal is a multi-tiered web application, it has multiple components that must be managed and maintained, adding to complexity and support costs.
+
+With the [end of life dates for Drupal versions 7 and 8 fast approaching](https://www.drupal.org/psa-2020-06-24), Drupal site owners should understand the options they have for migrating to new, simplified, more modern web platforms. This post will discuss migrating from Drupal to the Federalist platform, and will run through a simple example of how such a migration can be accomplished.
+
+This post will demonstrate how to use some of the commonly available Drupal migration tools to export content from a legacy Drupal site and import the content to a new Federalist starter template. We will discuss some of the issues that Drupal site owners should evaluate when considering a migration to a new platform.
+
+## Drupal 7 / 8 end of life
+
+The end of life date for both Drupal 7 and 8 is fast approaching. Drupal 8 wil reach it's scheduled end of life on November 2, 2021. Drupal 7 wil reach it's scheduled end of life on November 28, 2022 (after being extended for a period of about 1 year).
+
+As the challenges and work required to upgrade to the latest version of Drupal are considered by agencies, other platform options may also be evaluated. One potential option for current Drupal users is the [Federalist platform](https://federalist.18f.gov/) (soon to be renamed to [cloud.gov Pages](https://federalist.18f.gov/cloud-gov-pages/)).
```

Comment:
Change link to open in a new tab or window rather than refreshing the page.

Reply:
Done